db/migrations/20201109152715_add_screenshots_table.js

```diff
@@ -0,0 +1,23 @@
+const { addUpdateTsTrigger } = require("../util")
+
+exports.up = async (knex) => {
+  await knex.schema.createTable("screenshots", (table) => {
```

Comment:
Let's call this `feedback_images` and, at somepoint, rename this in other parts of the system as well